ext/pgsql/pgsql.stub.php

```diff
@@ -482,8 +482,8 @@ function pg_last_error(?PgSql\Connection $connection = null): string {}
 
     /**
      * @alias pg_last_error
-     * @deprecated
      */
+    #[\Deprecated]
     function pg_errormessage(?PgSql\Connection $connection = null): string {}
```

Comment:
All of these aliases have been deprecated since 8.0, possibly makes sense to indicate in the message to use the "proper" name which is indicated by the `@alias` doc annotation.

Except for the overloaded functions `pg_fetch_result()`, `pg_field_prtlen()`, and `pg_field_is_null()` which were deprecated in 8.4. cf. https://wiki.php.net/rfc/deprecate_functions_with_overloaded_signatures#pg_fetch_result_pg_field_prtlen_and_pg_field_is_null